net/relayd/src/opnsense/mvc/app/views/OPNsense/Relayd/index.volt

```diff
@@ -30,80 +30,106 @@ POSSIBILITY OF SUCH DAMAGE.
-	         'del':    '/api/relayd/settings/del/host/'
-	      });
+	      'search': '/api/relayd/settings/search/host/',
+	      'get':    '/api/relayd/settings/get/host/',
+	      'set':    '/api/relayd/settings/set/host/',
+	      'add':    '/api/relayd/settings/set/host/',
+	      'del':    '/api/relayd/settings/del/host/'
+	   });
 	   $("#grid-tablecheck").UIBootgrid({
-           'search': '/api/relayd/settings/search/tablecheck/',
-           'get':    '/api/relayd/settings/get/tablecheck/',
-           'set':    '/api/relayd/settings/set/tablecheck/',
-           'add':    '/api/relayd/settings/set/tablecheck/',
-           'del':    '/api/relayd/settings/del/tablecheck/'
-        });
+        'search': '/api/relayd/settings/search/tablecheck/',
+        'get':    '/api/relayd/settings/get/tablecheck/',
+        'set':    '/api/relayd/settings/set/tablecheck/',
+        'add':    '/api/relayd/settings/set/tablecheck/',
+        'del':    '/api/relayd/settings/del/tablecheck/'
+      });
 	   $("#grid-table").UIBootgrid({
-           'search': '/api/relayd/settings/search/table/',
-           'get':    '/api/relayd/settings/get/table/',
-           'set':    '/api/relayd/settings/set/table/',
-           'add':    '/api/relayd/settings/set/table/',
-           'del':    '/api/relayd/settings/del/table/'
-        });
+        'search': '/api/relayd/settings/search/table/',
+        'get':    '/api/relayd/settings/get/table/',
+        'set':    '/api/relayd/settings/set/table/',
+        'add':    '/api/relayd/settings/set/table/',
+        'del':    '/api/relayd/settings/del/table/'
+      });
 	   $("#grid-protocol").UIBootgrid({
-           'search': '/api/relayd/settings/search/protocol/',
-           'get':    '/api/relayd/settings/get/protocol/',
-           'set':    '/api/relayd/settings/set/protocol/',
-           'add':    '/api/relayd/settings/set/protocol/',
-           'del':    '/api/relayd/settings/del/protocol/'
-        });
+        'search': '/api/relayd/settings/search/protocol/',
+        'get':    '/api/relayd/settings/get/protocol/',
+        'set':    '/api/relayd/settings/set/protocol/',
+        'add':    '/api/relayd/settings/set/protocol/',
+        'del':    '/api/relayd/settings/del/protocol/'
+      });
 	   $("#grid-virtualserver").UIBootgrid({
-           'search': '/api/relayd/settings/search/virtualserver/',
-           'get':    '/api/relayd/settings/get/virtualserver/',
-           'set':    '/api/relayd/settings/set/virtualserver/',
-           'add':    '/api/relayd/settings/set/virtualserver/',
-           'del':    '/api/relayd/settings/del/virtualserver/'
-        });
+        'search': '/api/relayd/settings/search/virtualserver/',
+        'get':    '/api/relayd/settings/get/virtualserver/',
+        'set':    '/api/relayd/settings/set/virtualserver/',
+        'add':    '/api/relayd/settings/set/virtualserver/',
+        'del':    '/api/relayd/settings/del/virtualserver/'
+      });
```

Comment:
I am currently working on another plugin and I prefer a forEach iterator loop to keep the code short:
https://github.com/opnsense/plugins/pull/696/files#diff-ea6800eed0e9206f1e49a010e0e75ab9R85